text/0070-x-track-kusamanetwork.md

```diff
@@ -0,0 +1,160 @@
+agency to manage posts. It wouldn't be decentralized but it would probably be more effective in terms of creating good content.
+
+Finally, this solution is merely pseudo-decentralization since the X account manager would still have ultimate control of the account. It's decentralized insofar as
+the auth tokens are given to people actually running the tools; a house of cards is required to facilitate X posts via this track. Not ideal.
+
+## Testing, Security, and Privacy
+
+There's major precedent for configuring tracks on openGov given the amount of power tracks have, so it shouldn't be hard to come up with a sound configuration.
+That's why I recommend restricting permissions of this track to remarks and batches of remarks, or something equally inconsequential.
+
+Building the tools for this implementation is really straight-forward and could be audited by Fellowship members, and the community at large, on Github.
+
+The largest security concern would be the management of Kusama's X account's auth tokens. We would need to ensure that they aren't compromised.
+
+## Performance, Ergonomics, and Compatibility
+
+### Performance
+
+If a track on Kusama promises users that compliant referenda enacted therein would be posted on Kusama's X account, users would expect that track to perform as promised.
+If the house of cards tumbles down and a compliant referendum doesn't actually get anything posted, users might think that Kusama is broken or unreliable. This
+could be damaging to Kusama's image and cause people to question the soundness of other features on Kusama.
+
+As mentioned in the drawbacks, the performance of this feature would depend on off-chain coordinations. We can reduce the administrative burden of these coordinations
+by funding third parties with the Treasury to deal with it, but then we're relying on trusting these parties.
+
+### Ergonomics
+
+By adding a new track to Kusama, governance platforms like Polkassembly or Nova Wallet would need to include it on their applications. This shouldn't be too
+much of a burden or overhead since they've already built the infrastructure for other openGov tracks.
+
+### Compatibility
+
+This change wouldn't break any compatibility as far as I know.
+
+## References
+
+One reference to a similar feature requiring on-chain/off-chain coordination would be the Kappa-Sigma-Mu Society. Nothing on-chain necessarily enforces the rules
+or facilitates bids, challenges, defenses, etc. However, the Society has managed to maintain itself with integrity to its rules. So I don't think this is totally
+out of Kusama's scope. But it will require some off-chain effort to maintain.
+
+## Unresolved Questions
+
+- Who will develop the tools necessary to implement this feature? How do we select them?
+- How can this idea be better implemented with on-chain/substrate features?
```

Comment:
I think in the past i saw something from [Phala](https://phala.network/posts/web3-social-create-monetize-with-smart-contracts) that advertised to make social media posts from their blockchain. Dont know how it works.